Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -0,0 +1,118 @@
+//  Eatery
+//
+//  Created by William Ma on 9/1/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class BRBAccountViewController: UIViewController {
+    
+    private enum CellIdentifiers {
+        static let balance = "balance"
+        static let history = "history"
+    }
+    
+    private let account: BRBAccount
+    
+    private var tableView: UITableView!
+    
+    init(account: BRBAccount) {
+        self.account = account
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.register(BRBBalanceTableViewCell.self, forCellReuseIdentifier: CellIdentifiers.balance)
+        tableView.register(BRBHistoryTableViewCell.self, forCellReuseIdentifier: CellIdentifiers.history)
+        tableView.allowsSelection = false
+        tableView.separatorColor = .wash
+        tableView.dataSource = self
+        tableView.delegate = self
+        view.addSubview(tableView)
+        tableView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+
+}
+
+extension BRBAccountViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return 4
+        }
+        
+        return account.history.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.section == 0 {
+            let cell = tableView.dequeueReusableCell(withIdentifier: CellIdentifiers.balance) as! BRBBalanceTableViewCell
+            configure(balanceTableViewCell: cell, for: indexPath)
+            return cell
+        }
+
+        let cell = tableView.dequeueReusableCell(withIdentifier: CellIdentifiers.history) as! BRBHistoryTableViewCell
+        configure(historyTableViewCell: cell, for: indexPath)
+        return cell
+    }
+
+    private func configure(balanceTableViewCell cell: BRBBalanceTableViewCell, for indexPath: IndexPath) {
+        switch indexPath.row {
+        case 0:
+            cell.titleLabel.text = "Meal Swipes"
+            let swipesLeft = account.swipes == "" ? "Unlimited" : account.swipes
+            cell.subtitleLabel.text = swipesLeft
+        case 1:
+            cell.titleLabel.text = "BRBs"
+            cell.subtitleLabel.text = "$\(account.brbs)"
+        case 2:
+            cell.titleLabel.text = "Laundry"
+            cell.subtitleLabel.text = "$\(account.laundry)"
+        case 3:
+            cell.titleLabel.text = "City Bucks"
+            cell.subtitleLabel.text = "$\(account.cityBucks)"
```

Comment:
can we add a `func configure(with title: string, subtitle:string) {}` function to the cell and just pass these values in there instead?